amplicon-pooling.ipynb

```diff
@@ -1338,185 +1331,201 @@
+       "3       None  FWD:GTGYCAGCMGCCGCGGTAA; REV:GGACTACNVGGGTWTCTAAT   \n",
+       "4       None  FWD:GTGYCAGCMGCCGCGGTAA; REV:GGACTACNVGGGTWTCTAAT   \n",
        "\n",
-       "                                         pcr_primers center_name run_center  \\\n",
-       "0  FWD:GTGYCAGCMGCCGCGGTAA; REV:GGACTACNVGGGTWTCTAAT      UCSDMI     UCSDMI   \n",
-       "1  FWD:GTGYCAGCMGCCGCGGTAA; REV:GGACTACNVGGGTWTCTAAT      UCSDMI     UCSDMI   \n",
-       "2  FWD:GTGYCAGCMGCCGCGGTAA; REV:GGACTACNVGGGTWTCTAAT      UCSDMI     UCSDMI   \n",
-       "3  FWD:GTGYCAGCMGCCGCGGTAA; REV:GGACTACNVGGGTWTCTAAT      UCSDMI     UCSDMI   \n",
-       "4  FWD:GTGYCAGCMGCCGCGGTAA; REV:GGACTACNVGGGTWTCTAAT      UCSDMI     UCSDMI   \n",
+       "           sequencing_meth target_gene target_subfragment center_name  \\\n",
+       "0  Sequencing by synthesis    16S rRNA                 V4      UCSDMI   \n",
+       "1  Sequencing by synthesis    16S rRNA                 V4      UCSDMI   \n",
+       "2  Sequencing by synthesis    16S rRNA                 V4      UCSDMI   \n",
+       "3  Sequencing by synthesis    16S rRNA                 V4      UCSDMI   \n",
+       "4  Sequencing by synthesis    16S rRNA                 V4      UCSDMI   \n",
        "\n",
-       "   platform target_subfragment target_gene          sequencing_meth  \\\n",
-       "0  Illumina                 V4    16S rRNA  Sequencing by synthesis   \n",
-       "1  Illumina                 V4    16S rRNA  Sequencing by synthesis   \n",
-       "2  Illumina                 V4    16S rRNA  Sequencing by synthesis   \n",
-       "3  Illumina                 V4    16S rRNA  Sequencing by synthesis   \n",
-       "4  Illumina                 V4    16S rRNA  Sequencing by synthesis   \n",
+       "  center_project_name instrument_model runid  \n",
+       "0                None             None  None  \n",
+       "1                None             None  None  \n",
+       "2                None             None  None  \n",
+       "3                None             None  None  \n",
+       "4                None             None  None  \n",
        "\n",
-       "                       library_construction_protocol  \n",
-       "0  Illumina EMP protocol 515fbc, 806r amplificati...  \n",
-       "1  Illumina EMP protocol 515fbc, 806r amplificati...  \n",
-       "2  Illumina EMP protocol 515fbc, 806r amplificati...  \n",
-       "3  Illumina EMP protocol 515fbc, 806r amplificati...  \n",
-       "4  Illumina EMP protocol 515fbc, 806r amplificati...  \n",
-       "\n",
-       "[5 rows x 26 columns]"
+       "[5 rows x 34 columns]"
       ]
      },
-     "execution_count": 480,
+     "execution_count": 109,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
     "qiita_df = generate_qiita_prep_file(plate_df, seq_type)\n",
+    "\n",
+    "# Add eight additional columns to the prep-file that the wet-lab will fill in manually.\n",
```

Comment:
These changes should go in the generate_qiita_prep_file function that way you can add tests.

Reply:
Thanks @ElDeveloper ! I'll do that!